plugin/src/main/java/org/opensearch/ml/utils/RestActionUtils.java

```diff
@@ -87,13 +96,24 @@ public static FetchSourceContext getSourceContext(RestRequest request, SearchSou
         } else {
             // When user does not set the _source field in search model api request, searchSourceBuilder.fetchSource becomes null
             if (!userAgent.contains(OPENSEARCH_DASHBOARDS_USER_AGENT)) {
-                return new FetchSourceContext(true, Strings.EMPTY_ARRAY, UI_METADATA_EXCLUDE);
+                return new FetchSourceContext(true, Strings.EMPTY_ARRAY, ArrayUtils.add(UI_METADATA_EXCLUDE, "model_content"));
```

Comment:
Missed "content"?

Reply:
Sure let's add "content". I didn't add it in the first place because this old content field shouldn't be used at all. But just in case some old model still has it, let's add it.